src/api.js

```diff
@@ -13,9 +13,9 @@ var path          = require('path');
 var fs            = require('fs');
 require('superagent-hawk')(require('superagent'));
 var request       = require('superagent-promise');
-// Someone should rename utils to scopes... 
-var utils         = require('taskcluster-lib-scopes');
 var stats         = require('taskcluster-lib-stats');
+var scopes        = require('taskcluster-lib-scopes');
+var monitoring    = require('taskcluster-lib-monitor');
```

Comment:
Why scopes or why monitoring? scopes I just changed because I saw a comment above this that said `// Someone should rename utils to scopes... `.